thewalrus/_hafnian.py

```diff
@@ -178,6 +180,45 @@ def hafnian(
     )
 
 
+def hafnian_sparse(A, D: set = None):
```

Comment:
```suggestion
def hafnian_sparse(A, D = None):
```

Reply:
why?

Reply:
none of the functions in this file does that.

Reply:
I thought it was an improvement!